.gitignore

```diff
@@ -1,6 +1,7 @@
 # Created by .ignore support plugin (hsz.mobi)
 ### Node template
 # Logs
+package-lock.json
```

Comment:
We need to commit package-lock.json. Refer to this: https://docs.npmjs.com/files/package-lock.json